draw/draw_empress_wrapper.py

```diff
@@ -0,0 +1,136 @@
+# This file is not run by unittest by default
+# Run this file to generate drawing examples
+
+import empress
+import matplotlib.pyplot as plt
+from pathlib import Path
+from draw import common
+
+common.create_output_folder()
+
+example_input_path = "./examples/test-size5-no924.newick"
```

Comment:
I suggest that we move away from .newick files.  This is the "old" input format.  We're now moving to having three input files comprising a host tree (.nwk), parasite tree (.nwk), and mapping (.mapping).  It's easy to create these from a .newick file:  The first tree in the .newick file goes in the host's .nwk file, the second one goes in the parasite's .nwk file, and the remaining stuff goes in the .mapping file.  Since this is the way that we plan to request user input, it makes sense for our tests to use this too.

Reply:
Yes. We should change this in Zach's pull request which will change the one file input to three files input.